src/udiskslinuxencrypted.c

```diff
@@ -438,8 +514,20 @@ handle_unlock (UDisksEncrypted        *encrypted,
                                              "Error unlocking %s: %s",
                                              udisks_block_get_device (block),
                                              error->message);
+
+      /* Restore the old encryption type if the unlock failed, because
+       * in this case we don't know for sure if we used the correct
+       * encryption type. */
+      udisks_encrypted_set_hint_encryption_type(encrypted, old_hint_encryption_type);
+
       goto out;
     }
+  else
+    {
+      /* We have to free old_hint_encryption_type if and only if it was not
+       * used in udisks_encrypted_set_hint_encryption_type() */
+      g_free(old_hint_encryption_type);
```

Comment:
Missing space again (and also above).

Reply:
Also fixed in 932aff4be696d43ad2a7b4117713443e93b23b05.